src/components/ArticleList.jsx

```diff
@@ -0,0 +1,71 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../utils/api";
+import LoadingDisplay from "./minor/LoadingDisplay";
+import ErrorDisplay from "./minor/ErrorDisplay";
+
+// ArticleList Component
+// Renders all articles found at /api/articles, with a limit of 10 per page.
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(0);
+
+  useEffect(() => {
+    const fetchArticles = async () => {
+      try {
+        const data = await getArticles(currentPage);
+        setArticles(data.articles);
+        setTotalPages(Math.ceil(data.total_count / 10));
+        setIsLoading(false);
+      } catch (err) {
+        setError(
+          "There was an error fetching the articles.  Please try again."
+        );
+        setIsLoading(false);
+      }
+    };
+
+    fetchArticles();
+  }, [currentPage]);
+
+  if (isLoading) return <LoadingDisplay />;
+  if (error) return <ErrorDisplay />;
+
+  return (
+    <div className="grid gap-5" aria-label="Articles">
```

Comment:
this would be more semantic if you had it as a ul and then li - for the list of articles